hal/armv7m/imxrt/117x/imxrt.h

```diff
@@ -645,6 +645,14 @@ enum { clk_pllarm = 0, clk_pllsys1, clk_pllsys2, clk_pllsys3, clk_pllaudio, clk_
 /* clang-format on */
 
 
+typedef struct {
+	unsigned char enabled;
+	unsigned char platformEnabled;
+	unsigned int timeoutMs;
+	unsigned int defaultTimeoutMs;
+} _imxrt_wdgInfo_t;
```

Comment:
That's unseen naming convention - we use `_` prefix to mark functions that need outside locking, it's hard to tell what it means in case of a data structure (as it's always locked externally, also this one does not need locking at all).